csharp-basic-topics/AwaitInForEachLoops/Tests/TaskWhenAllTest.cs

```diff
@@ -0,0 +1,27 @@
+﻿
+
+using AwaitInForEachLoop;
+
+namespace Tests
+{
+    [TestFixture]
+    public class TaskWhenAllInLoopTests
+    {
+        [Test]
+        public async Task WhenResultAsync_ThenCompleteProcessingTasks()
+        {
```

Comment:
Update the test case with respect to the code changes done as per above feedback